scipy/sparse/linalg/_special_sparse_arrays.py

```diff
@@ -51,7 +49,7 @@ class LaplacianNd(LinearOperator):
 
     The Laplacian matrix of a graph (`scipy.sparse.csgraph.laplacian`) of a
     rectangular grid corresponds to the negative Laplacian with the Neumann
-    conditions, i.e., ``boundary_conditions = 'neumann'``.
+    conditions, i.e., ``boundary_conditions = "neumann"``.
```

Comment:
These seem unnecessary and unrelated to the rest of this PR.

Reply:
99% of those who heard of the "Laplacian" nowadays mean the graph Laplacian. After being asked about the distinction many times, I want the connection to be clear - the actual reason for this example that I thus would like to keep.

Reply:
I was referring to the double-quotes changes.

Reply:
Done